changelog/8178.bugfix.md

```diff
@@ -0,0 +1 @@
+Caught ChannelNotFoundEntity exception and re-raised as RasaException.
```

Comment:
This needs to be updated. Also might be worth mentioning the error is coming from the pika broker.